src/pages/explore/dex/_components/ExploreDTokensSection.tsx

```diff
@@ -0,0 +1,214 @@
+    [symbol: string]: DexPrice;
+  }>();
+
+  const [cryptoTokens, setCryptoTokens] = useState<{
+    [symbol: string]: DexPrice;
+  }>();
+  const [dstocksTokens, setDstocksTokens] = useState<{
+    [symbol: string]: DexPrice;
+  }>();
+  const [etfsTokens, setEtfsTokens] = useState<{
+    [symbol: string]: DexPrice;
+  }>();
+
+  const tokenNameMap = {
+    DUSD: t("exploreDTokensSection.tokenNames.DUSD"),
+    BTC: t("exploreDTokensSection.tokenNames.BTC"),
+    ETH: t("exploreDTokensSection.tokenNames.ETH"),
+    USDC: t("exploreDTokensSection.tokenNames.USDC"),
+    EUROC: t("exploreDTokensSection.tokenNames.EUROC"),
+    USDT: t("exploreDTokensSection.tokenNames.USDT"),
+    LTC: t("exploreDTokensSection.tokenNames.LTC"),
+    BCH: t("exploreDTokensSection.tokenNames.BCH"),
+    TSLA: t("exploreDTokensSection.tokenNames.TSLA"),
+    BABA: t("exploreDTokensSection.tokenNames.BABA"),
+    GOOGL: t("exploreDTokensSection.tokenNames.GOOGL"),
+    AAPL: t("exploreDTokensSection.tokenNames.AAPL"),
+    NVDA: t("exploreDTokensSection.tokenNames.NVDA"),
+    MSFT: t("exploreDTokensSection.tokenNames.MFST"),
+    PYPL: t("exploreDTokensSection.tokenNames.PYPL"),
+    ARKK: t("exploreDTokensSection.tokenNames.ARKK"),
+    SPY: t("exploreDTokensSection.tokenNames.SPY"),
+    GLD: t("exploreDTokensSection.tokenNames.GLD"),
+    VNQ: t("exploreDTokensSection.tokenNames.VNQ"),
+    SLV: t("exploreDTokensSection.tokenNames.SLV"),
+    URTH: t("exploreDTokensSection.tokenNames.URTH"),
+    QQQ: t("exploreDTokensSection.tokenNames.QQQ"),
+  };
+
+  useEffect(() => {
+    api.poolpairs.listDexPrices("USDT").then((res) => {
+      setTokens(
+        tokenNameMap,
+        res,
+        setCryptoTokens,
+        setDstocksTokens,
+        setEtfsTokens,
+        setSelectedTabTokens
+      );
+    });
+  }, [api.poolpairs]);
+
+  useEffect(() => {
+    switch (tokenSectionChoice) {
+      case TokenLabels.ETFS:
+        setSelectedTabTokens(etfsTokens);
+        break;
+      case TokenLabels.DSTOCKS:
+        setSelectedTabTokens(dstocksTokens);
+        break;
+      case TokenLabels.CRYPTO:
+      default:
+        setSelectedTabTokens(cryptoTokens);
+    }
+  }, [tokenSectionChoice]);
+
+  const dimension = useWindowDimensions();
+
+  return (
+    <Container className="mb-[240px]">
+      <div className="flex flex-col md:items-center items-start">
+        <SectionTitle text={t("exploreDTokensSection.title")} />
+        <div
+          className={classNames(
+            "lg:text-[60px] lg:leading-[72px] mt-5 lg:tracking-normal",
+            "text-[40px] leading-[44px] tracking-[-0.02em]"
+          )}
+        >
+          {t("exploreDTokensSection.subtitle")}
+        </div>
+
+        <ExploreDTokensTabs
+          tokenSectionChoice={tokenSectionChoice}
+          setTokenSectionChoice={setTokenSectionChoice}
+        />
+        <ExploreDTokensGrid selectionTokens={selectedTabTokens} />
+
+        <SecondaryButton
```

Comment:
should navigate to defiscan tokens page on click

Reply:
click box bigger than my house 🌚
<img width="325" alt="image" src="https://user-images.githubusercontent.com/40191153/229711477-35c7acf5-1acf-461c-abc0-664e53017a73.png">